client/flutter/lib/pages/onboarding/notifications_page.dart

```diff
@@ -23,90 +23,111 @@ class _NotificationsPageState extends State<NotificationsPage> {
   Widget build(BuildContext context) {
     return Material(
       color: CupertinoColors.white,
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.stretch,
-        children: <Widget>[
-          Expanded(
-            child: Container(
-              alignment: Alignment.bottomLeft,
-              padding: EdgeInsets.only(left: 36.0, bottom: 18.0),
-              child: SvgPicture.asset(
-                  'assets/svg/undraw-onboarding-notifications.svg'),
-            ),
-          ),
-          Column(
+      child: Container(
```

Comment:
I believe since we have a Material with a background color we can delete the container from the heirarchy.

Reply:
you're right, i removed it in my last commit